src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -116,6 +116,12 @@ protected override void OnNodeRemoved(NodeModel node)
             EngineController.NodeDeleted(node);
         }
 
+        protected override void ResetWorkspaceCore()
+        {
+            // Reset Run Automatic option to false on resetting the workspace
+            DynamicRunEnabled = false;
+        }
+
```

Comment:
Hi @aparajit-pratap, the reason this is added now is because we want no-auto run by default? What if you don't set this?